From 8110ebd55d5bf659cd40dab2df59d80dafdb367a Mon Sep 17 00:00:00 2001
From: Kevin Layer <layer@franz.com>
Date: Mon, 10 Feb 2014 11:10:42 -0800
Subject: [PATCH 2/2] fix bugs in rfe12668 implementation

The previous commit added :embed and :keygen but they were already
there.  Resolve this issue.

Change-Id: Ieb962a12880394e67d973835945005446833fab7
---
 aserve/htmlgen/htmlgen.cl | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/aserve/htmlgen/htmlgen.cl b/aserve/htmlgen/htmlgen.cl
index 0c0d6e8..f77d6de 100644
--- a/aserve/htmlgen/htmlgen.cl
+++ b/aserve/htmlgen/htmlgen.cl
@@ -658,7 +658,6 @@
 (def-std-html :dt        t nil)
 
 (def-std-html :em        t nil)
-(def-std-html :embed     t nil)
 
 (def-std-html :fieldset        t nil)
 (def-std-html :font        t nil)
@@ -685,7 +684,6 @@
 (def-std-html :isindex    nil nil)
 
 (def-std-html :kbd  	t nil)
-(def-std-html :keygen  	nil nil)
 
 (def-std-html :label  	t nil)
 (def-std-html :layer  	t nil)
@@ -782,5 +780,5 @@
 (def-std-html :menuitem	 	t nil)
 (def-std-html :details	 	t nil)
 (def-std-html :datalist	 	t nil)
-(def-std-html :keygen	 	t nil)
+(def-std-html :keygen	 	nil nil)
 (def-std-html :output	 	t nil)
-- 
2.25.1

